index.js

```diff
@@ -3,7 +3,16 @@ const escapeStringRegexp = require('escape-string-regexp');
 
 const reCache = new Map();
 
-function makeRe(pattern, shouldNegate) {
+global.options = {
```

Comment:
Remove

Reply:
@SamVerschueren I never saw `global` before so did some investigation and have a question for you. Node [documentation](https://nodejs.org/api/globals.html#globals_global) says that 

> ... In Node.js ... The top-level scope is not the global scope; var something inside a Node.js module will be local to that module.

If we use `global.`, would that mean that there's potential clash between `options`, let's say if couple other modules were using same `options` and same `caseSensitive` key? If `global.options` really sets the object to the global scope, then is it better if we kept options within the top level of the **module's** scope?